src/main/java/bio/terra/service/snapshotbuilder/SnapshotBuilderService.java

```diff
@@ -83,7 +83,9 @@ public SnapshotBuilderGetConceptsResponse getConceptChildren(
       UUID datasetId, Integer conceptId, AuthenticatedUserRequest userRequest) {
     Dataset dataset = datasetService.retrieve(datasetId);
     TableNameGenerator tableNameGenerator = getTableNameGenerator(dataset, userRequest);
-    String cloudSpecificSql = buildConceptChildrenQuery(conceptId, tableNameGenerator);
+    String cloudSpecificSql =
+        buildConceptChildrenQuery(
```

Comment:
Can you change this file so it doesn't static import this method (and `buildSearchConceptsQuery`) directly? Statically imported methods make it harder to see what's going on, since it makes it look like it's a method call within the current class.
```suggestion
        ConceptChildrenQueryBuilder.buildConceptChildrenQuery(
```

Once you're using the class name, you could make the method name shorter as now it's clear from the class name which query is being built:
```suggestion
        ConceptChildrenQueryBuilder.buildQuery(
```